packages/teuchos/numerics/src/Teuchos_SerialSymDenseMatrix.hpp

```diff
@@ -397,10 +397,14 @@ class SerialSymDenseMatrix : public CompObject, public BLAS<OrdinalType,ScalarTy
                  const OrdinalType numCols)
   {
     const size_t size = size_t(numRows) * size_t(numCols);
+#ifndef _WIN32
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wvla"
+#endif
     return new ScalarType[size];
+#ifndef _WIN32
 #pragma GCC diagnostic pop
+#endif
```

Comment:
Same as above, just strip out these `#pragma` statements.  There should be no need for them with any C++ standard version.